### ISSUE TITLE
Python 2

### ISSUE THREAD

**Issue by drwolff** [BishopFox/spoofcheck#9]
This tool is written in Python 2 which is depreciated. It still works, but users need to specify pip2 when installing the requirements and python2 when executing the script.

**Comment by fsacer** [comment]
Agreed I mentioned the issue here too https://github.com/lunarca/pyemailprotectionslib/pull/6
